build.gradle

```diff
@@ -58,7 +58,7 @@ repositories {
 publishing {
     publications {
         mavenJava(MavenPublication) {
-            groupId = "software.amazon.smithy"
+            groupId = "com.disneystreaming.oss"
```

Comment:
That's the org in my version of the extension. Should I change it to com.disneystreaming.smithy?

Reply:
yeah